megamek/src/megamek/server/Server.java

```diff
@@ -37490,20 +36129,22 @@ private void registerWithServerBrowser(boolean register, String urlString) {
                     }
                 }
             } else {
+                // TODO : Windchild Can this be removed?
```

Comment:
This does nothing, can it just be removed?

Reply:
That looks like old debugging code. I don't see any reason to keep it.

Reply:
Found a few other places with obvious debugging code, removed them too.